apps/web/tailwind.config.js

```diff
@@ -19,15 +18,24 @@ module.exports = {
       'venice-black-drop-shadow': `0px 2px 4px 0px ${VeniceTheme.dropShadow}`,
     },
     extend: {
-      backgroundColor: {
-        'venice-black': VeniceTheme.black,
+      colors: {
+        ...VeniceTheme,
+        'venice-black': {
+          DEFAULT: VeniceTheme.black,
+          300: '#4e4e4e',
+          500: '#2e2e2e',
+        },
       },
-      colors: VeniceTheme,
+      current: 'currentColor',
```

Comment:
Needs to enable this to support `fill-current`, so we can style SVG icon colors using the container/parent style.